README.md

```diff
@@ -10,17 +10,31 @@ Test via ```nosetests```.
 ### Usage: 
     POST   on /shopcarts: creates new shopcart.
         Expects JSON body data:
-            item_id: -1
+            item_id: 0
             user_id: int
+            item_name: string, "ring" + str(item_id)
+            quantity: int, range from 0 to 999
+            price: float, range from 0.0 to 0.1
+    GET    on /shopcarts: returns all of the shopcarts.
+    GET    on /shopcarts/<user-id>: returns items in <user-id> shopcart.
+    DELETE on /shopcarts/<user-id>: deletes <user-id> shopcart.
+    POST   on /shopcarts/<user-id>/items: creates new item in <user-id> shopcart.
+        Expects JSON body data:
+            item_id: int
+            quantity: int 
             item_name: string
-            quantity: int
             price: float
-    GET    on /shopcarts: returns list of all shopcarts
```

Comment:
This route is still there, shouldn't be deleted

Reply:
Just location changes, never delete. Thanks Timothy. Sorry for so many errors.

Reply:
Thank you for fixing!